MultiMystery.py

```diff
@@ -180,14 +183,20 @@ def get_alive_threads():
                             tqdm.write("YAML error")
                             break
                         done = check_if_done()
-                        if done:
+                        if done and not keep_all_seeds:
                             break
                     else:
                         msg = f"Seed Attempt #{task.task_id:4} was successful."
 
                         dead_or_alive[task.task_id] = True
+                        alive[task.task_id] = True
                         done = check_if_done()
-                        if done:
+                        if keep_all_seeds:
+                            tqdm.write(msg)
+                            os.makedirs(os.path.join(output_path, basedir, str(task.task_id)))
+                            for file in os.listdir(task.folder.name):
+                                shutil.copy(os.path.join(task.folder.name, file), os.path.join(output_path, basedir, str(task.task_id), file))
```

Comment:
If we're having two places where this code is needed (copying out a working seed and packaging it), it should be moved in a function instead of copied.

Reply:
made a copy_seed function, which copies the seed, and returns its seedname